adapters/Mintegral/MintegralAdapter/waterfall/GADMAdapterMintegralBannerAdLoader.m

```diff
@@ -0,0 +1,141 @@
+// Copyright 2022 Google LLC
```

Comment:
For all new files added, they should be `2023`.

Reply:
ok,I have modified.